tests/integration/components/add-reminder-test.js

```diff
@@ -0,0 +1,25 @@
+import { moduleForComponent, test, skip } from 'ember-qunit';
```

Comment:
This is a boilerplate test. If you're going to keep this file make sure you have a legitimate assertion.